src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -813,6 +814,30 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
     );
   };
 
+  isLoginForButtonFeatureEnabled = () => {
+    if (this.props.buttonConfig.loginForButtonFeature) {
+      return this.goToLogin();
+    }
+    return false;
+  }
+
+  goToLogin = () => {
+    if (!this.props.user || !this.props.user.username) {
+      this.props.history.push('/login');
```

Comment:
```suggestion
      this.props.history.push('/login', { from: `${this.props.location.pathname}` });
```
So it can get back to this page after login

But you will need to add `location` as props of this component